src/StoreValue.ts

```diff
@@ -0,0 +1,97 @@
+import urltemplate from 'url-template'
+import { isTemplatedLink, isEntityReference, isCollection } from './halHelpers.js'
+import QueryablePromise from './QueryablePromise'
+import EmbeddedCollection from './EmbeddedCollection.js'
+import CanHaveItems from './CanHaveItems.js'
+import Resource from './interfaces/Resource'
+import ApiActions from './interfaces/ApiActions'
+import StoreData from './interfaces/StoreData'
+import StoreValueCreator from './StoreValueCreator.js'
+import { InternalConfig } from './interfaces/Config.js'
+
+/**
+ * Creates an actual StoreValue, by wrapping the given Vuex store storeData. The storeData must not be loading.
+ * If the storeData has been loaded into the store before but is currently reloading, the old storeData will be
+ * returned, along with a ._meta.load promise that resolves when the reload is complete.
+ * @param storeData fully loaded entity storeData from the Vuex store
+ */
+class StoreValue extends CanHaveItems implements Resource {
+  public _meta: {
+    self: string,
+    load: Promise<Resource>
+  }
+
+  private storeData: StoreData
+  config: InternalConfig
+  apiActions: ApiActions
+
+  constructor (storeData: StoreData, { get, reload, post, patch, del, isUnknown }: ApiActions, storeValueCreator: StoreValueCreator, config: InternalConfig) {
+    super({ get, reload, isUnknown }, config)
+
+    this.apiActions = { get, reload, post, patch, del, isUnknown }
+    this.config = config
+    this.storeData = storeData
+
+    Object.keys(storeData).forEach(key => {
+      const value = storeData[key]
+
+      // TODO: Why is the next line needed.? Store data should never contain a property 'allItems'. Or can it?
+      if (key === 'allItems' && isCollection(storeData)) return
```

Comment:
I agree, can you remove the line and check if the tests still run?
This was added when `allItems` was [introduced](https://github.com/ecamp/ecamp3/commit/7ec5a67ff7e9226f4bf044f00e89e345e1a422db), might have been a debugging leftover